pkg/controller/clusterrole/clusterrole_controller.go

```diff
@@ -0,0 +1,113 @@
+package clusterrole
```

Comment:
So you meant I could have generated the controllers as well along with the operator stuff. And then filled it out in this commit. Well, I could still do it but it will take some work. Can you review this as is for now.

Reply:
Yeah totally, don't worry about it, just for future.